.circleci/config.yml

```diff
@@ -171,23 +165,18 @@ jobs:
   #                 done
 
   publish-docker-tag:
-    executor: docker-publisher
+    executor: linuxgo
     steps:
       - buildevents/with_job_span:
           steps:
             - checkout
             - setup_remote_docker
+            - run: go install github.com/google/ko@latest
             - run:
-                name: Build Docker image
-                command: docker build -t $IMAGE_NAME:latest .
-            - run:
-                name: Publish Docker Image to Docker Hub
+                name: "Publish multi-arch docker image"
                 command: |
-                  echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
-                  IMAGE_TAG=${CIRCLE_TAG/v/''}
-                  docker tag $IMAGE_NAME:latest $IMAGE_NAME:$IMAGE_TAG
-                  docker push $IMAGE_NAME:latest
-                  docker push $IMAGE_NAME:$IMAGE_TAG
+                  echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin;
+                  VERSION=${CIRCLE_TAG} KO_DOCKER_REPO=honeycombio build/build_docker.sh
```

Comment:
I'm not sure if KO_DOCKER_REPO=honeycombio is correct, or should be the FQDN + repo name. Guess we'll find out.

(My own usage of ko has been to ECR, not dockerhub)

Reply:
My read of [the ko README](https://github.com/google/ko#choose-destination) has me thinking the Docker Hub username by itself is correct when publishing to the Hub of Dockers.

Reply:
Yes, that's correct - when you set that the image is named like `{dockername}/{project}` which is correct for publishing to Docker Hub 👍🏻